plugins/woocommerce-blocks/tests/e2e/tests/filter-blocks/price-filter.block_theme.spec.ts

```diff
@@ -0,0 +1,175 @@
+			);
+
+			// Min price input field
+			const leftInputContainer = page.locator(
+				'.wp-block-woocommerce-product-filter-price-content-left-input'
+			);
+			const minPriceInput = leftInputContainer.locator( '.min' );
+			const minPriceInputValue = await minPriceInput.inputValue();
+
+			expect( minPriceInputValue ).toBe( '$20' );
+
+			// Min price slider thumb
+			const priceSlider = page.locator(
+				'.wp-block-woocommerce-product-filter-price-content-price-range-slider'
+			);
+			const minPriceThumb = priceSlider.locator( 'input[name="min"]' );
+			const minPriceThumbValue = await minPriceThumb.inputValue();
+
+			expect( minPriceThumbValue ).toBe( '20' );
+
+			// Max price input field
+			const rightInputContainer = page.locator(
+				'.wp-block-woocommerce-product-filter-price-content-right-input'
+			);
+			const maxPriceInput = rightInputContainer.locator( '.max' );
+			const maxPriceInputValue = await maxPriceInput.inputValue();
+
+			expect( maxPriceInputValue ).toBe( '$67' );
+
+			// Max price slider thumb
+			const maxPriceThumb = priceSlider.locator( 'input[name="max"]' );
+			const maxPriceThumbValue = await maxPriceThumb.inputValue();
+
+			expect( maxPriceThumbValue ).toBe( '67' );
+		} );
+
+		test( 'Changes in the price input field triggers price slider updates', async ( {
+			page,
+			defaultBlockPost,
+		} ) => {
+			await page.goto(
+				`${ defaultBlockPost.link }?min_price=20&max_price=67`
+			);
+
+			// Min price input field
+			const leftInputContainer = page.locator(
+				'.wp-block-woocommerce-product-filter-price-content-left-input'
+			);
+			const minPriceInput = leftInputContainer.locator( '.min' );
+			await minPriceInput.fill( '30' );
+			await minPriceInput.blur();
+			const minPriceInputValue = await minPriceInput.inputValue();
+
+			expect( minPriceInputValue ).toBe( '$30' );
+
+			// Min price slider thumb
+			const priceSlider = page.locator(
+				'.wp-block-woocommerce-product-filter-price-content-price-range-slider'
+			);
+			const minPriceThumb = priceSlider.locator( 'input[name="min"]' );
+			const minPriceThumbValue = await minPriceThumb.inputValue();
+
+			expect( minPriceThumbValue ).toBe( '30' );
+
+			// Max price input field
+			const rightInputContainer = page.locator(
+				'.wp-block-woocommerce-product-filter-price-content-right-input'
+			);
+			const maxPriceInput = rightInputContainer.locator( '.max' );
+			await maxPriceInput.fill( '80' );
+			await maxPriceInput.blur();
+			const maxPriceInputValue = await maxPriceInput.inputValue();
+
+			expect( maxPriceInputValue ).toBe( '$80' );
+
+			// Max price slider thumb
+			const maxPriceThumb = priceSlider.locator( 'input[name="max"]' );
+			const maxPriceThumbValue = await maxPriceThumb.inputValue();
+
+			expect( maxPriceThumbValue ).toBe( '80' );
+		} );
+
+		test( 'Price input field rejects min price higher than max price', async ( {
+			page,
+			defaultBlockPost,
+		} ) => {
+			const maxPrice = 67;
```

Comment:
It looks like this variable isn't needed. although if you wanted to you could use it [in the assertion](https://github.com/woocommerce/woocommerce/pull/45855/files#diff-524ce0ba17c55755ebb47ef00ad631b056f0739fcdfa1c659f61d6ddfa74c0a2R142) by casting it to a string?

Reply:
You're correct. Initially, my idea was to reuse it multiple times, but then I realized it wouldn't be as readable, and I forgot to remove it. I fixed this in: https://github.com/woocommerce/woocommerce/pull/45855/commits/cbf7df1cb13b6d7de13fdf3fdc9f5c4fb26b8a83